frontend/src/queries/nodes/InsightQuery/utils/queryNodeToFilter.ts

```diff
@@ -142,7 +154,7 @@ export const queryNodeToFilter = (query: InsightQueryNode): Partial<FilterType>
         sampling_factor: query.samplingFactor,
     })
 
-    if (!isRetentionQuery(query) && !isPathsQuery(query)) {
+    if (!isRetentionQuery(query) && !isPathsQuery(query) && 'series' in query) {
```

Comment:
For most of the stuff on this PR, if you don't like it, I can add those other fields to the WebAnalytics query schemas.